backend/app/Recommendation/Application/Repositories/Eloquent/RecommendationRepository.php

```diff
@@ -14,11 +15,34 @@ public function getAll(): array
         return RecommendationEloquentModel::all()->toArray();
     }
 
+    /**
+     * @throws \Exception
+     */
     public function store(Recommendation $recommendation)
     {
+
         $recommendationEloquent = RecommendationMapper::toEloquent($recommendation);
         $recommendationEloquent->save();
+        $recommendationId = $recommendationEloquent->id;
+
+        array_map(function ($answer) use ($recommendationId) {
+            $AnswerEloquentModel = AnswerMapper::toEloquent($answer);
+            $AnswerEloquentModel->recommendation_id = $recommendationId;
+            $AnswerEloquentModel->save();
+        }, $recommendation->getAnswerSeparateDate() );
```

Comment:
Давай завернем в `foreach`

```suggestion
        foreach ($recommendation->getAnswerSeparateData() as $answer) {
            $answerEloquent = AnswerMapper::toEloquent($answer);
            $answerEloquent->recommendation_id = $recommendationId;
            $answerEloquent->save();
        }
```